corehq/toggles/__init__.py

```diff
@@ -901,6 +901,13 @@ def _ensure_valid_randomness(randomness):
     help_link='https://confluence.dimagi.com/display/saas/USH%3A+Show+current+domain+in+web+apps+Login+As+banner',
 )
 
+WEB_APPS_UPLOAD_QUESTIONS = StaticToggle(
```

Comment:
Since you plan to release this soon you could use a `FeatureRelease` to make that clear.

Reply:
cb3aac6b011